Makefile.am

```diff
@@ -224,6 +223,15 @@ bdist_wheel: $(WHEELDISTDIR)
 wheel_bundle: $(WHEELBUNDLEDIR) bdist_wheel
 	$(PYTHON) -m pip wheel --wheel-dir $(WHEELBUNDLEDIR) $(WHEELDISTDIR)/*.whl
 
+wheel_placeholder: $(WHEELDISTDIR)
+	for dir in $(IPA_PLACEHOLDERS); do \
+	    $(MAKE) $(AM_MAKEFLAGS) -C $(top_srcdir)/pypi/$${dir} bdist_wheel || exit 1; \
+	done
+
+pypi_packages: bdist_wheel wheel_placeholder
+	@echo -e "\n\nTo upload packages to PyPI, run:\n"
+	@echo -e "    twine upload $(WHEELDISTDIR)/*-$(VERSION)-py2.py3-none-any.whl\n"
+
```

Comment:
that could be solved with
```
bdist_wheet:
    $(MAKE) -C $(top_builddir)/pypi
```

It would make the main Makefile nicer+simpler but that's just nice to have

The main problem is not in this PR but in structure of directories in freeipa git.
There are mixed directories for Python code/C-code/javascript code.
As a result of this there are many *.*SUBDIR* variables in top level makefile to workaround this directory structure.

I thought that moving some *.*SUBDIR* variables to `pypi/Makefile.am` would simplify it.

 Sorry for a little noise.